bin/behat-prepare.sh

```diff
@@ -66,3 +66,18 @@ git push
 terminus wp "core install --title=$TERMINUS_ENV-$TERMINUS_SITE --url=$PANTHEON_SITE_URL --admin_user=pantheon --admin_email=wp-redis@getpantheon.com --admin_password=pantheon"
 terminus wp "cache flush"
 terminus wp "plugin activate wp-redis"
+
+###
+# Download the Pantheon WordPress Upstream tests
+###
+cd $BASH_DIR/..
```

Comment:
Let's move the downloading of these tests to `require-dev` of composer.json. Behat itself is already there.